src/app/pages/user-team-create-page/user-team-create-page.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnInit, Renderer } from "@angular/core";
+import { Router } from "@angular/router";
+import { AuthService } from "angular-aap-auth";
+
+// Import Services.
+import { UserService } from "../../services/user.service";
+import { TeamsService } from "../../services/teams.service";
+import { EndpointService } from "../../services/endpoint.service";
+import { RequestsService } from "../../services/requests.service";
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { environment } from '../../../environments/environment';
+
+@Component({
+  selector: "app-user-team-create-page",
+  templateUrl: "./user-team-create-page.component.html",
+  styleUrls: ["./user-team-create-page.component.scss"],
+  providers: [
+    AuthService,
+    UserService,
+    TeamsService,
+    EndpointService,
+    RequestsService
+  ]
+})
+export class UserTeamCreatePageComponent implements OnInit {
+  teams = {};
+  private tokenListener: Function;
+  token: string;
+  userteamsEndpoint = "";
+  teamCreateForm : FormGroup;
+  constructor(
+    public renderer: Renderer,
+    public authService: AuthService,
+    public endpointService: EndpointService,
+    public requestsService: RequestsService,
+    public router: Router,
+  ) {}
+
+  async ngOnInit() {
+    this.teamCreateForm = new FormGroup({
+      centreName: new FormControl('', Validators.required),
+      description: new FormControl(''),
```

Comment:
Is the `description` field not required? I think it is. So, we should add a validator to it.

Reply:
Description field is not required in the API.

Reply:
Well, then we should not require it on the 'html' page, either.
Please, remove it from there.

Reply:
I left the html markup as its sort of general validation markup. It will appears only for field with required validators but its good to add different validation messages for all the fields even if its not being used for time being. What do you think?

Reply:
I am sorry, but that kind of unneeded code is called dead code. The code should be as simple as it could be. The unneeded code is just 'noise' around the useful code.
Could you remove those lines, please?

Reply:
No problem, I've removed the markup.

Reply:
Thanks